core/app/components/forms/elements/FIleUploadElement.tsx

```diff
@@ -1,10 +1,12 @@
 "use client";
 
 import dynamic from "next/dynamic";
+import { Value } from "@sinclair/typebox/value";
```

Comment:
the file name has a capital I `FIle`

Reply:
shockingly hard problem to solve on a mac lol